product_brand/product_brand_view.xml

```diff
@@ -152,10 +153,10 @@
                 <field name="name" position="after">
                     <field name="product_brand_id"/>
                 </field>
-                <group string='Group by...' position="inside">
+<!--                <group string='Group by...' position="inside">
```

Comment:
Use `<group posititon="inside">` instead. It will find the only group in the search